Failing tests on Arch Linux (Go 1.16.5)

Hi,

I try to build rekor right now and there are a few failing tests.

Do I need to generate something special prior build? (Note: I don't use the makefile..)
Right now I am building rekor-cli and rekor-server like this:

```
  export CGO_CPPFLAGS="${CPPFLAGS}"
  export CGO_CFLAGS="${CFLAGS}"
  export CGO_CXXFLAGS="${CXXFLAGS}"
  export CGO_LDFLAGS="${LDFLAGS}"
  export GOFLAGS="-buildmode=pie -trimpath -mod=readonly -modcacherw"
  go build -ldflags "-linkmode=external -w" -o cmd/rekor-cli/rekor-cli cmd/rekor-cli/main.go
  go build -ldflags "-linkmode=external -w" -o cmd/rekor-server/rekor-server cmd/rekor-server/main.go
```

Here is the full log with `go test -v ./...`:

```
go: downloading go.uber.org/goleak v1.1.10
go: downloading github.com/shibumi/go-pathspec v1.2.0
?   	github.com/sigstore/rekor/cmd/rekor-cli	[no test files]
=== RUN   TestArtifactPFlags
--- PASS: TestArtifactPFlags (0.02s)
=== RUN   TestValidateRekorServerURL
--- PASS: TestValidateRekorServerURL (0.00s)
=== RUN   TestSearchPFlags
--- PASS: TestSearchPFlags (0.00s)
=== RUN   TestAPIKey
--- PASS: TestAPIKey (0.00s)
=== RUN   TestTimestampFlags
--- PASS: TestTimestampFlags (0.00s)
PASS
ok  	github.com/sigstore/rekor/cmd/rekor-cli/app	0.038s
?   	github.com/sigstore/rekor/cmd/rekor-cli/app/format	[no test files]
?   	github.com/sigstore/rekor/cmd/rekor-cli/app/state	[no test files]
?   	github.com/sigstore/rekor/cmd/rekor-server	[no test files]
?   	github.com/sigstore/rekor/cmd/rekor-server/app	[no test files]
?   	github.com/sigstore/rekor/pkg/api	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/client	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/client/entries	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/client/index	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/client/operations	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/client/pubkey	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/client/timestamp	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/client/tlog	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/models	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/restapi	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/restapi/operations	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/restapi/operations/entries	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/restapi/operations/index	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/restapi/operations/pubkey	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/restapi/operations/timestamp	[no test files]
?   	github.com/sigstore/rekor/pkg/generated/restapi/operations/tlog	[no test files]
?   	github.com/sigstore/rekor/pkg/log	[no test files]
=== RUN   TestFactoryNewKey
=== RUN   TestFactoryNewKey/valid_pgp
=== RUN   TestFactoryNewKey/valid_minisign
=== RUN   TestFactoryNewKey/valid_x509
=== RUN   TestFactoryNewKey/valid_ssh
=== RUN   TestFactoryNewKey/invalid_ssh_signature
=== RUN   TestFactoryNewKey/invalid_ssh_key
=== RUN   TestFactoryNewKey/#00
--- PASS: TestFactoryNewKey (0.00s)
    --- PASS: TestFactoryNewKey/valid_pgp (0.00s)
    --- PASS: TestFactoryNewKey/valid_minisign (0.00s)
    --- PASS: TestFactoryNewKey/valid_x509 (0.00s)
    --- PASS: TestFactoryNewKey/valid_ssh (0.00s)
    --- PASS: TestFactoryNewKey/invalid_ssh_signature (0.00s)
    --- PASS: TestFactoryNewKey/invalid_ssh_key (0.00s)
    --- PASS: TestFactoryNewKey/#00 (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/pki	0.007s
=== RUN   TestReadPublicKey
--- PASS: TestReadPublicKey (0.00s)
=== RUN   TestReadSignature
--- PASS: TestReadSignature (0.00s)
=== RUN   TestReadErrorPublicKey
--- PASS: TestReadErrorPublicKey (0.00s)
=== RUN   TestReadErrorSignature
--- PASS: TestReadErrorSignature (0.00s)
=== RUN   TestCanonicalValueSignature
--- PASS: TestCanonicalValueSignature (0.00s)
=== RUN   TestCanonicalValuePublicKey
--- PASS: TestCanonicalValuePublicKey (0.00s)
=== RUN   TestVerifySignature
--- PASS: TestVerifySignature (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/pki/minisign	0.005s
=== RUN   TestReadPublicKey
--- PASS: TestReadPublicKey (0.01s)
=== RUN   TestReadSignature
--- PASS: TestReadSignature (0.00s)
=== RUN   TestReadErrorPublicKey
--- PASS: TestReadErrorPublicKey (0.00s)
=== RUN   TestReadErrorSignature
--- PASS: TestReadErrorSignature (0.00s)
=== RUN   TestFetchPublicKey
--- PASS: TestFetchPublicKey (0.01s)
=== RUN   TestFetchSignature
--- PASS: TestFetchSignature (0.00s)
=== RUN   TestCanonicalValueSignature
--- PASS: TestCanonicalValueSignature (0.00s)
=== RUN   TestCanonicalValuePublicKey
--- PASS: TestCanonicalValuePublicKey (0.00s)
=== RUN   TestEmailAddresses
--- PASS: TestEmailAddresses (0.01s)
=== RUN   TestVerifySignature
--- PASS: TestVerifySignature (0.01s)
PASS
ok  	github.com/sigstore/rekor/pkg/pki/pgp	0.045s
=== RUN   TestSignature_Verify
=== RUN   TestSignature_Verify/ec
--- PASS: TestSignature_Verify (0.01s)
    --- PASS: TestSignature_Verify/ec (0.01s)
=== RUN   TestSignature_VerifyFail
=== RUN   TestSignature_VerifyFail/ec
--- PASS: TestSignature_VerifyFail (0.00s)
    --- PASS: TestSignature_VerifyFail/ec (0.00s)
=== RUN   TestEmailAddresses
=== RUN   TestEmailAddresses/ec
=== RUN   TestEmailAddresses/email
--- PASS: TestEmailAddresses (0.00s)
    --- PASS: TestEmailAddresses/ec (0.00s)
    --- PASS: TestEmailAddresses/email (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/pki/pkcs7	0.012s
=== RUN   TestFromOpenSSH
=== RUN   TestFromOpenSSH/rsa
    sign_test.go:156: cmd ssh-keygen -Y sign -n file -f /tmp/TestFromOpenSSH_rsa2470759055/001/id /tmp/TestFromOpenSSH_rsa2470759055/001/data:
    sign_test.go:156: exec: "ssh-keygen": executable file not found in $PATH
=== RUN   TestFromOpenSSH/ed25519
    sign_test.go:156: cmd ssh-keygen -Y sign -n file -f /tmp/TestFromOpenSSH_ed255191234061598/001/id /tmp/TestFromOpenSSH_ed255191234061598/001/data:
    sign_test.go:156: exec: "ssh-keygen": executable file not found in $PATH
--- FAIL: TestFromOpenSSH (0.00s)
    --- FAIL: TestFromOpenSSH/rsa (0.00s)
    --- FAIL: TestFromOpenSSH/ed25519 (0.00s)
=== RUN   TestToOpenSSH
=== RUN   TestToOpenSSH/rsa
    sign_test.go:217: cmd ssh-keygen -Y verify -f /tmp/TestToOpenSSH_rsa3887725137/001/allowed_signer -I test@rekor.dev -n file -s /tmp/TestToOpenSSH_rsa3887725137/001/oursig:
    sign_test.go:217: exec: "ssh-keygen": executable file not found in $PATH
=== RUN   TestToOpenSSH/ed25519
    sign_test.go:217: cmd ssh-keygen -Y verify -f /tmp/TestToOpenSSH_ed255192342998196/001/allowed_signer -I test@rekor.dev -n file -s /tmp/TestToOpenSSH_ed255192342998196/001/oursig:
    sign_test.go:217: exec: "ssh-keygen": executable file not found in $PATH
--- FAIL: TestToOpenSSH (0.01s)
    --- FAIL: TestToOpenSSH/rsa (0.00s)
    --- FAIL: TestToOpenSSH/ed25519 (0.00s)
=== RUN   TestRoundTrip
=== RUN   TestRoundTrip/rsa
=== RUN   TestRoundTrip/ed25519
--- PASS: TestRoundTrip (0.01s)
    --- PASS: TestRoundTrip/rsa (0.01s)
    --- PASS: TestRoundTrip/ed25519 (0.00s)
FAIL
FAIL	github.com/sigstore/rekor/pkg/pki/ssh	0.020s
=== RUN   TestSignature_Verify
=== RUN   TestSignature_Verify/rsa
=== RUN   TestSignature_Verify/ec
=== RUN   TestSignature_Verify/ed25519
--- PASS: TestSignature_Verify (0.01s)
    --- PASS: TestSignature_Verify/rsa (0.00s)
    --- PASS: TestSignature_Verify/ec (0.01s)
    --- PASS: TestSignature_Verify/ed25519 (0.00s)
=== RUN   TestSignature_VerifyFail
=== RUN   TestSignature_VerifyFail/rsa
=== RUN   TestSignature_VerifyFail/ec
=== RUN   TestSignature_VerifyFail/ed25519
--- PASS: TestSignature_VerifyFail (0.00s)
    --- PASS: TestSignature_VerifyFail/rsa (0.00s)
    --- PASS: TestSignature_VerifyFail/ec (0.00s)
    --- PASS: TestSignature_VerifyFail/ed25519 (0.00s)
=== RUN   TestNilCertChainToPEM
--- PASS: TestNilCertChainToPEM (0.00s)
=== RUN   TestCertChain_Verify
--- PASS: TestCertChain_Verify (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/pki/x509	0.021s
=== RUN   TestMemory
--- PASS: TestMemory (0.02s)
PASS
ok  	github.com/sigstore/rekor/pkg/signer	0.028s
?   	github.com/sigstore/rekor/pkg/storage	[no test files]
=== RUN   TestNewEntry
--- PASS: TestNewEntry (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/types	0.010s
=== RUN   TestIntotoType
2021-07-14T01:34:21.368Z	ERROR	types/versionmap.go:82	Could not get version from string: "not"
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).SetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:82
github.com/sigstore/rekor/pkg/types/intoto.TestIntotoType
	github.com/sigstore/rekor/pkg/types/intoto/intoto_test.go:90
testing.tRunner
	testing/testing.go:1193
2021-07-14T01:34:21.368Z	ERROR	types/versionmap.go:58	No Major.Minor.Patch elements found
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).GetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:58
github.com/sigstore/rekor/pkg/types.(*RekorType).VersionedUnmarshal
	github.com/sigstore/rekor/pkg/types/types.go:50
github.com/sigstore/rekor/pkg/types/intoto.BaseIntotoType.UnmarshalEntry
	github.com/sigstore/rekor/pkg/types/intoto/intoto.go:56
github.com/sigstore/rekor/pkg/types/intoto.TestIntotoType
	github.com/sigstore/rekor/pkg/types/intoto/intoto_test.go:125
testing.tRunner
	testing/testing.go:1193
--- PASS: TestIntotoType (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/intoto	0.009s
=== RUN   TestNewEntryReturnType
--- PASS: TestNewEntryReturnType (0.00s)
=== RUN   TestV001Entry_Unmarshal
=== RUN   TestV001Entry_Unmarshal/empty
=== RUN   TestV001Entry_Unmarshal/missing_envelope
=== RUN   TestV001Entry_Unmarshal/missing_envelope#01
=== RUN   TestV001Entry_Unmarshal/valid
=== RUN   TestV001Entry_Unmarshal/invalid
=== RUN   TestV001Entry_Unmarshal/invalid_key
--- PASS: TestV001Entry_Unmarshal (0.01s)
    --- PASS: TestV001Entry_Unmarshal/empty (0.00s)
    --- PASS: TestV001Entry_Unmarshal/missing_envelope (0.00s)
    --- PASS: TestV001Entry_Unmarshal/missing_envelope#01 (0.00s)
    --- PASS: TestV001Entry_Unmarshal/valid (0.00s)
    --- PASS: TestV001Entry_Unmarshal/invalid (0.00s)
    --- PASS: TestV001Entry_Unmarshal/invalid_key (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/intoto/v0.0.1	0.020s
=== RUN   TestJARType
2021-07-14T01:34:21.365Z	ERROR	types/versionmap.go:82	Could not get version from string: "not"
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).SetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:82
github.com/sigstore/rekor/pkg/types/jar.TestJARType
	github.com/sigstore/rekor/pkg/types/jar/jar_test.go:89
testing.tRunner
	testing/testing.go:1193
2021-07-14T01:34:21.365Z	ERROR	types/versionmap.go:58	No Major.Minor.Patch elements found
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).GetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:58
github.com/sigstore/rekor/pkg/types.(*RekorType).VersionedUnmarshal
	github.com/sigstore/rekor/pkg/types/types.go:50
github.com/sigstore/rekor/pkg/types/jar.(*BaseJARType).UnmarshalEntry
	github.com/sigstore/rekor/pkg/types/jar/jar.go:56
github.com/sigstore/rekor/pkg/types/jar.TestJARType
	github.com/sigstore/rekor/pkg/types/jar/jar_test.go:124
testing.tRunner
	testing/testing.go:1193
--- PASS: TestJARType (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/jar	0.009s
=== RUN   TestNewEntryReturnType
--- PASS: TestNewEntryReturnType (0.00s)
=== RUN   TestCrossFieldValidation
--- PASS: TestCrossFieldValidation (0.02s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/jar/v0.0.1	0.029s
=== RUN   TestRekordType
2021-07-14T01:34:21.364Z	ERROR	types/versionmap.go:82	Could not get version from string: "not"
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).SetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:82
github.com/sigstore/rekor/pkg/types/rekord.TestRekordType
	github.com/sigstore/rekor/pkg/types/rekord/rekord_test.go:90
testing.tRunner
	testing/testing.go:1193
2021-07-14T01:34:21.364Z	ERROR	types/versionmap.go:58	No Major.Minor.Patch elements found
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).GetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:58
github.com/sigstore/rekor/pkg/types.(*RekorType).VersionedUnmarshal
	github.com/sigstore/rekor/pkg/types/types.go:50
github.com/sigstore/rekor/pkg/types/rekord.BaseRekordType.UnmarshalEntry
	github.com/sigstore/rekor/pkg/types/rekord/rekord.go:56
github.com/sigstore/rekor/pkg/types/rekord.TestRekordType
	github.com/sigstore/rekor/pkg/types/rekord/rekord_test.go:125
testing.tRunner
	testing/testing.go:1193
--- PASS: TestRekordType (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/rekord	0.009s
=== RUN   TestNewEntryReturnType
--- PASS: TestNewEntryReturnType (0.00s)
=== RUN   TestCrossFieldValidation
--- PASS: TestCrossFieldValidation (0.02s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/rekord/v0.0.1	0.026s
=== RUN   TestRfc3161Type
2021-07-14T01:34:21.368Z	ERROR	types/versionmap.go:82	Could not get version from string: "not"
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).SetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:82
github.com/sigstore/rekor/pkg/types/rfc3161.TestRfc3161Type
	github.com/sigstore/rekor/pkg/types/rfc3161/rfc3161_test.go:94
testing.tRunner
	testing/testing.go:1193
2021-07-14T01:34:21.368Z	ERROR	types/versionmap.go:58	No Major.Minor.Patch elements found
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).GetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:58
github.com/sigstore/rekor/pkg/types.(*RekorType).VersionedUnmarshal
	github.com/sigstore/rekor/pkg/types/types.go:50
github.com/sigstore/rekor/pkg/types/rfc3161.BaseTimestampType.UnmarshalEntry
	github.com/sigstore/rekor/pkg/types/rfc3161/rfc3161.go:56
github.com/sigstore/rekor/pkg/types/rfc3161.TestRfc3161Type
	github.com/sigstore/rekor/pkg/types/rfc3161/rfc3161_test.go:129
testing.tRunner
	testing/testing.go:1193
--- PASS: TestRfc3161Type (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/rfc3161	0.009s
=== RUN   TestNewEntryReturnType
--- PASS: TestNewEntryReturnType (0.00s)
=== RUN   TestCrossFieldValidation
--- PASS: TestCrossFieldValidation (0.01s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/rfc3161/v0.0.1	0.022s
=== RUN   TestRPMType
2021-07-14T01:34:21.365Z	ERROR	types/versionmap.go:82	Could not get version from string: "not"
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).SetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:82
github.com/sigstore/rekor/pkg/types/rpm.TestRPMType
	github.com/sigstore/rekor/pkg/types/rpm/rpm_test.go:90
testing.tRunner
	testing/testing.go:1193
2021-07-14T01:34:21.365Z	ERROR	types/versionmap.go:58	No Major.Minor.Patch elements found
github.com/sigstore/rekor/pkg/types.(*SemVerEntryFactoryMap).GetEntryFactory
	github.com/sigstore/rekor/pkg/types/versionmap.go:58
github.com/sigstore/rekor/pkg/types.(*RekorType).VersionedUnmarshal
	github.com/sigstore/rekor/pkg/types/types.go:50
github.com/sigstore/rekor/pkg/types/rpm.(*BaseRPMType).UnmarshalEntry
	github.com/sigstore/rekor/pkg/types/rpm/rpm.go:56
github.com/sigstore/rekor/pkg/types/rpm.TestRPMType
	github.com/sigstore/rekor/pkg/types/rpm/rpm_test.go:125
testing.tRunner
	testing/testing.go:1193
--- PASS: TestRPMType (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/rpm	0.010s
=== RUN   TestNewEntryReturnType
--- PASS: TestNewEntryReturnType (0.00s)
=== RUN   TestCrossFieldValidation
--- PASS: TestCrossFieldValidation (0.02s)
PASS
ok  	github.com/sigstore/rekor/pkg/types/rpm/v0.0.1	0.038s
=== RUN   TestMarshalCheckpoint
=== RUN   TestMarshalCheckpoint/bananas
=== RUN   TestMarshalCheckpoint/the_view_from_the_tree_tops_is_great!
=== RUN   TestMarshalCheckpoint/the_view_from_the_tree_tops_is_great!#01
--- PASS: TestMarshalCheckpoint (0.00s)
    --- PASS: TestMarshalCheckpoint/bananas (0.00s)
    --- PASS: TestMarshalCheckpoint/the_view_from_the_tree_tops_is_great! (0.00s)
    --- PASS: TestMarshalCheckpoint/the_view_from_the_tree_tops_is_great!#01 (0.00s)
=== RUN   TestUnmarshalCheckpoint
=== RUN   TestUnmarshalCheckpoint/valid_one
=== RUN   TestUnmarshalCheckpoint/valid_with_different_ecosystem
=== RUN   TestUnmarshalCheckpoint/valid_with_trailing_data
=== RUN   TestUnmarshalCheckpoint/valid_with_multiple_trailing_data_lines
=== RUN   TestUnmarshalCheckpoint/valid_with_trailing_newlines
=== RUN   TestUnmarshalCheckpoint/invalid_-_insufficient_lines
=== RUN   TestUnmarshalCheckpoint/invalid_-_empty_header
=== RUN   TestUnmarshalCheckpoint/invalid_-_missing_newline_on_roothash
=== RUN   TestUnmarshalCheckpoint/invalid_size_-_not_a_number
=== RUN   TestUnmarshalCheckpoint/invalid_size_-_negative
=== RUN   TestUnmarshalCheckpoint/invalid_size_-_too_large
=== RUN   TestUnmarshalCheckpoint/invalid_roothash_-_not_base64
--- PASS: TestUnmarshalCheckpoint (0.00s)
    --- PASS: TestUnmarshalCheckpoint/valid_one (0.00s)
    --- PASS: TestUnmarshalCheckpoint/valid_with_different_ecosystem (0.00s)
    --- PASS: TestUnmarshalCheckpoint/valid_with_trailing_data (0.00s)
    --- PASS: TestUnmarshalCheckpoint/valid_with_multiple_trailing_data_lines (0.00s)
    --- PASS: TestUnmarshalCheckpoint/valid_with_trailing_newlines (0.00s)
    --- PASS: TestUnmarshalCheckpoint/invalid_-_insufficient_lines (0.00s)
    --- PASS: TestUnmarshalCheckpoint/invalid_-_empty_header (0.00s)
    --- PASS: TestUnmarshalCheckpoint/invalid_-_missing_newline_on_roothash (0.00s)
    --- PASS: TestUnmarshalCheckpoint/invalid_size_-_not_a_number (0.00s)
    --- PASS: TestUnmarshalCheckpoint/invalid_size_-_negative (0.00s)
    --- PASS: TestUnmarshalCheckpoint/invalid_size_-_too_large (0.00s)
    --- PASS: TestUnmarshalCheckpoint/invalid_roothash_-_not_base64 (0.00s)
=== RUN   TestSigningRoundtripCheckpoint
=== RUN   TestSigningRoundtripCheckpoint/Log_Checkpoint_RSA_v0
=== RUN   TestSigningRoundtripCheckpoint/Log_Checkpoint_ECDSA_v0
=== RUN   TestSigningRoundtripCheckpoint/Log_Checkpoint_Ed25519_v0
=== RUN   TestSigningRoundtripCheckpoint/Log_Checkpoint_Mismatch_v0
=== RUN   TestSigningRoundtripCheckpoint/Log_Checkpoint_Mismatch_v1
=== RUN   TestSigningRoundtripCheckpoint/Log_Checkpoint_Mismatch_v2
=== RUN   TestSigningRoundtripCheckpoint/Log_Checkpoint_Mismatch_v3
--- PASS: TestSigningRoundtripCheckpoint (0.21s)
    --- PASS: TestSigningRoundtripCheckpoint/Log_Checkpoint_RSA_v0 (0.00s)
    --- PASS: TestSigningRoundtripCheckpoint/Log_Checkpoint_ECDSA_v0 (0.00s)
    --- PASS: TestSigningRoundtripCheckpoint/Log_Checkpoint_Ed25519_v0 (0.00s)
    --- PASS: TestSigningRoundtripCheckpoint/Log_Checkpoint_Mismatch_v0 (0.00s)
    --- PASS: TestSigningRoundtripCheckpoint/Log_Checkpoint_Mismatch_v1 (0.00s)
    --- PASS: TestSigningRoundtripCheckpoint/Log_Checkpoint_Mismatch_v2 (0.00s)
    --- PASS: TestSigningRoundtripCheckpoint/Log_Checkpoint_Mismatch_v3 (0.00s)
=== RUN   TestInvalidSigVerification
=== RUN   TestInvalidSigVerification/Log_Checkpoint_v0
=== RUN   TestInvalidSigVerification/Log_Checkpoint_v0#01
=== RUN   TestInvalidSigVerification/Log_Checkpoint_v0#02
=== RUN   TestInvalidSigVerification/Log_Checkpoint_Ed25519_v0
--- PASS: TestInvalidSigVerification (0.00s)
    --- PASS: TestInvalidSigVerification/Log_Checkpoint_v0 (0.00s)
    --- PASS: TestInvalidSigVerification/Log_Checkpoint_v0#01 (0.00s)
    --- PASS: TestInvalidSigVerification/Log_Checkpoint_v0#02 (0.00s)
    --- PASS: TestInvalidSigVerification/Log_Checkpoint_Ed25519_v0 (0.00s)
=== RUN   TestUnmarshalSignedCheckpoint
=== RUN   TestUnmarshalSignedCheckpoint/invalid_checkpoint,_no_signatures
=== RUN   TestUnmarshalSignedCheckpoint/valid_checkpoint,_no_signatures
=== RUN   TestUnmarshalSignedCheckpoint/incorrect_signature_line_format
=== RUN   TestUnmarshalSignedCheckpoint/signature_not_base64_encoded
=== RUN   TestUnmarshalSignedCheckpoint/missing_identity
=== RUN   TestUnmarshalSignedCheckpoint/signature_base64_encoded_but_too_short
=== RUN   TestUnmarshalSignedCheckpoint/valid_signed_checkpoint_-_single_signature
=== RUN   TestUnmarshalSignedCheckpoint/valid_signed_checkpoint_-_two_signatures
--- PASS: TestUnmarshalSignedCheckpoint (0.00s)
    --- PASS: TestUnmarshalSignedCheckpoint/invalid_checkpoint,_no_signatures (0.00s)
    --- PASS: TestUnmarshalSignedCheckpoint/valid_checkpoint,_no_signatures (0.00s)
    --- PASS: TestUnmarshalSignedCheckpoint/incorrect_signature_line_format (0.00s)
    --- PASS: TestUnmarshalSignedCheckpoint/signature_not_base64_encoded (0.00s)
    --- PASS: TestUnmarshalSignedCheckpoint/missing_identity (0.00s)
    --- PASS: TestUnmarshalSignedCheckpoint/signature_base64_encoded_but_too_short (0.00s)
    --- PASS: TestUnmarshalSignedCheckpoint/valid_signed_checkpoint_-_single_signature (0.00s)
    --- PASS: TestUnmarshalSignedCheckpoint/valid_signed_checkpoint_-_two_signatures (0.00s)
=== RUN   TestCreateTimestampRequest
--- PASS: TestCreateTimestampRequest (0.00s)
=== RUN   TestParseTimestampRequest
--- PASS: TestParseTimestampRequest (0.00s)
=== RUN   TestCreateRFC3161Response
--- PASS: TestCreateRFC3161Response (0.00s)
PASS
ok  	github.com/sigstore/rekor/pkg/util	0.233s
FAIL
```

Ah ok! I missed it, my bad. The unit tests shell out to "ssh-keygen" to verify signature compatibility back and forth:

```shell
    sign_test.go:156: cmd ssh-keygen -Y sign -n file -f /tmp/TestFromOpenSSH_rsa2470759055/001/id /tmp/TestFromOpenSSH_rsa2470759055/001/data:
    sign_test.go:156: exec: "ssh-keygen": executable file not found in $PATH
=== RUN   TestFromOpenSSH/ed25519
    sign_test.go:156: cmd ssh-keygen -Y sign -n file -f /tmp/TestFromOpenSSH_ed255191234061598/001/id /tmp/TestFromOpenSSH_ed255191234061598/001/data:
    sign_test.go:156: exec: "ssh-keygen": executable file not found in $PATH
--- FAIL: TestFromOpenSSH (0.00s)
```

We should probably document that as a test dependency and fail with a SKIP if it's not found.

--- a/pkg/pki/ssh/sign_test.go
+++ b/pkg/pki/ssh/sign_test.go
@@ -140,6 +140,9 @@ func TestFromOpenSSH(t *testing.T) {
 			priv: ed25519PrivateKey,
 		},
 	} {
+		if _, err := exec.LookPath("ssh-keygen"); err != nil {
+			t.Skip("skip TestFromOpenSSH: missing ssh-keygen in PATH")
+		}
 		t.Run(tt.name, func(t *testing.T) {
 			tt := tt
 
@@ -193,6 +196,9 @@ func TestToOpenSSH(t *testing.T) {
 			priv: ed25519PrivateKey,
 		},
 	} {
+		if _, err := exec.LookPath("ssh-keygen"); err != nil {
+			t.Skip("skip TestToOpenSSH: missing ssh-keygen in PATH")
+		}
 		t.Run(tt.name, func(t *testing.T) {
 			tt := tt
 			// Test that a signature from here can validate in the CLI.

